src/Databases/removeWhereConditionPlaceholder.cpp

```diff
@@ -0,0 +1,24 @@
+#include <Databases/removeWhereConditionPlaceholder.h>
+
+namespace DB
+{
+
+std::string removeWhereConditionPlaceholder(const std::string  & query)
+{
+    /** In case UPDATE_FIELD is specified in {condition} for dictionary that must load all data.
+      * Replace {condition} with true_condition for initial dictionary load.
+      * For next dictionary loads {condition} will be updated with UPDATE_FIELD.
+      */
```

Comment:
Maybe move this comment to .h file and pretend it is description? (The function is undocumented and its name tells me not much.)